src/cli/cli.cpp

```diff
@@ -3468,8 +3468,8 @@ otError Interpreter::ProcessRouter(uint8_t aArgsLength, char *aArgs[])
 
         if (isTable)
         {
-            OutputLine("| ID | RLOC16 | Next Hop | Path Cost | LQ In | LQ Out | Age | Extended MAC     |");
-            OutputLine("+----+--------+----------+-----------+-------+--------+-----+------------------+");
+            OutputLine("| ID | RLOC16 | Next Hop | Path Cost | LQ In | LQ Out | Age | Extended MAC     | Link |");
```

Comment:
Does this mean bidirectional link or unidirectional link?
```suggestion
            OutputLine("| ID | RLOC16 | Next Hop | Path Cost | LQ In | LQ Out | Age | Extended MAC     | Linked |");
```

Reply:
Agree that `Link` is not a perfect header name. 
But I used `Link` because `router <id>` command is already using `Link: `

See https://github.com/openthread/openthread/blob/master/src/cli/README.md#router-id for reference.

Reply:
I will keep it as is to be inline with `router <id>` command.